docs/kyma/04-03-cluster-installation.md

```diff
@@ -193,75 +140,41 @@ Set the following environment variables:
     ```
```

Comment:
This entire option should be moved to 04-04 

* If you install Kyma from sources, check out [kyma-project](https://github.com/kyma-project/kyma) and enter the root folder. Run: